m2cgen/interpreters/vba/interpreter.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from m2cgen import ast
+from m2cgen.interpreters import mixins, utils
+from m2cgen.interpreters.interpreter import ToCodeInterpreter
+from m2cgen.interpreters.vba.code_generator import VbaCodeGenerator
+
+
+class VbaInterpreter(ToCodeInterpreter, mixins.LinearAlgebraMixin):
+    supported_bin_vector_ops = {
+        ast.BinNumOpType.ADD: "addVectors",
+    }
+
+    supported_bin_vector_num_ops = {
+        ast.BinNumOpType.MUL: "mulVectorNumber",
+    }
+
+    exponent_function_name = "Exp"
+    power_function_name = "Application.WorksheetFunction.Power"
+    tanh_function_name = "Application.WorksheetFunction.Tanh"
+
+    def __init__(self, indent=4, *args, **kwargs):
+        cg = VbaCodeGenerator(indent=indent)
+        kwargs["feature_array_name"] = "input_vector"
```

Comment:
I would like to strongly recommend changing default value to something else, because `input` is a reserved word in many languages. For instance, here, for VBA this arg name results in invalid syntax.